stdlib/public/runtime/Metadata.cpp

```diff
@@ -5655,10 +5656,8 @@ void swift::verifyMangledNameRoundtrip(const Metadata *metadata) {
   // variable lets us easily turn on verification to find and fix these
   // bugs. Remove this and leave it permanently on once everything works
   // with it enabled.
-  bool verificationEnabled =
```

Comment:
Remove the last sentence of the comment above, since we no longer plan to remove this verification code.

Reply:
Good idea. I'll do that in a followup, since I just got the thing through PR testing and I'd rather not continue that particular struggle....